vendor/rakwireless/rak4631-868-profile.yaml

```diff
@@ -0,0 +1,8 @@
+vendorProfileID: 4631
+supportsClassB: false
+supportsClassC: false
+macVersion: 1.0.2
+regionalParametersVersion: RP001-1.0.2
+supportsJoin: true
+maxEIRP: 22
```

Comment:
This is too high for EU868 region

Reply:
Thanks Johan, I wasn't sure if that was what the hardware was capable of or as limited by the firmware for the region.
Fixed now.